pkg/kubernetes/subsetdiff.go

```diff
@@ -93,60 +92,55 @@ func subsetDiff(c client.Client, m manifest.Manifest) (*difference, error) {
 		return nil, errors.Wrap(err, "getting state from cluster")
 	}
 
-	should, err := yaml.Marshal(m)
-	if err != nil {
-		return nil, err
-	}
+	should := m.String()
 
-	is, err := yaml.Marshal(subset(m, rawIs))
-	if err != nil {
-		return nil, err
-	}
-	if string(is) == "{}\n" {
-		is = []byte("")
+	sub := subset(m, rawIs)
```

Comment:
Where is rawIs set? From what I can see it is only an empty MSI here.

Reply:
https://github.com/grafana/tanka/blob/2ee719f219079a91c8f03a39bc2a17399eab2874/pkg/kubernetes/subsetdiff.go#L82-L87